package.json

```diff
@@ -88,8 +90,10 @@
     "webpack-serve": "^2.0.3"
   },
   "files": [
-    "/dash_core_components/*{.js,.map}"
+    "/dash_core_components/*{.js,.map}",
+    "/lib/**"
   ],
+  "sideEffects": false,
```

Comment:
Declare this package side-effect free. Let's tools know they can tree-shake without problems.